components/user/EventRow.tsx

```diff
@@ -193,6 +225,14 @@ const summarizeEvent = (
         <Verbose>Promoted on {hostname}</Verbose>
       </>
     )
+  } else if (type === EventType.NODE_HOSTED) {
+    return false
```

Comment:
Maybe just return an empty react fragment for this if you don't want to deal with handling the boolean? I haven't looked at the implementation too closely.